README.rst

```diff
@@ -134,6 +148,13 @@ Font
 -  **ttf** - ``application/font-sfnt``
 -  **otf** - ``application/font-sfnt``
 
+Document
```

Comment:
Group microsoft office pre 2007 documents as `application/x-ole-storage` as this 3 types have same file signatures. We can determine the file type by filename extension.